objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/neuralNet/NeuralNetInterface2.java

```diff
@@ -0,0 +1,44 @@
+package io.vantiq.extsrc.objectRecognition.neuralNet;
+
+import java.util.Map;
+
+import io.vantiq.extsrc.objectRecognition.exception.FatalImageException;
+import io.vantiq.extsrc.objectRecognition.exception.ImageProcessingException;
+
+/**
+ * An interface for the neural net that will process the image and return a List of data representing the objects found.
+ */
+public interface NeuralNetInterface2 extends NeuralNetInterface {
+    /**
```

Comment:
Define constant for IMAGE_TIMESTAMP `public static final String IMAGE_TIMESTAMP = imageTimestamp;`.

Avoid "magic strings" in code.  Also tells implementors of the interface what they're expected to deal with/do.